components/director/pkg/operation/api.go

```diff
@@ -128,21 +132,39 @@ func prepareLastOperation(resource model.Entity) *OperationResponse {
 		Error: resource.GetError(),
 	}
 
+	prepareOperationType(resource, opResponse)
```

Comment:
Maybe rename the `prepareLastOperation` function into `buildLastOperation`. Also maybe move the `prepareOperationType`, `prepareOperationStatus` and `prepareCreationTime` functions as methods to the `OperationResponse` and rename them to something like `initializeOperationType`, `initializeOperationStatus`, `initializeCreationTime` or if you have a better word than **prepare** or **initialize**.